scrapscript.py

```diff
@@ -561,6 +567,9 @@ def deserialize(msg: Dict[str, object]) -> "String":
         assert isinstance(msg["value"], str)
         return String(msg["value"])
 
+    def __str__(self) -> str:
```

Comment:
This will break if it has quotes so maybe we should have a note-to-self or TODO about that

Reply:
Don't fully understand how and when this breaks, but added a comment. Thanks for catching